code/game/objects/items/dehy_carp.dm

```diff
@@ -39,12 +39,13 @@
 /obj/item/toy/carpplushie/dehy_carp/proc/Swell()
 	desc = "It's growing!"
 	visible_message("<span class='notice'>[src] swells up!</span>")
-
 	// Animation
 	icon = 'icons/mob/carp.dmi'
 	flick("carp_swell", src)
 	// Wait for animation to end
-	sleep(6)
+	addtimer(CALLBACK(src, .proc/Spawn), 6)
+
+/obj/item/toy/carpplushie/dehy_carp/proc/Spawn()
```

Comment:
this needs to be named differently

[DM Ref - spawn()](http://www.byond.com/docs/ref/#/proc/spawn)